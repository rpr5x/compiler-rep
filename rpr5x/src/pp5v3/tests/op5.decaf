int d;
void main()
{
  int a;
  bool b;
  int c;
  d = 5 + 20 * - - 10;
  Print(5 + 10 % 7 * 8 + a * d, d, a, 10 * 7 /3);
  b = 5 < (4 * 10 + -5 - 100);
  Print(b || (0 != a && 4 == (2 * 2)), !(4 == 5));
}