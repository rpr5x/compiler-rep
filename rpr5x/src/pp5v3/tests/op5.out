Loaded: /if14/ww6r/WorkingDisk/TA/compiler/exceptions/exceptions.s
29205023falsetrue